web-client/src/presenter/computeds/dashboardExternalHelper.test.js

```diff
@@ -36,50 +36,65 @@ describe('petitioner dashboard helper', () => {
     });
     expect(result.showCaseList).toEqual(true);
     expect(result.showWhatToExpect).toEqual(false);
-    expect(result.showCaseSearch).toEqual(false);
   });
-  it('shows case search if defined user has privatePractitioner role', () => {
+
+  it('should keep the showFileACase flag as false when the user role is not a private practitioner', () => {
```

Comment:
Hm, for this `it(...)` statement, might me want to explicit for being a `petitioner` or loop through `petitioner` and  `irsrPractitioner`?

Reply:
add an roles array and looped through it

Reply:
changed it block wording to "...when the user role is petitioner"